src/OpenApi/gen/XmlCommentGenerator.Emitter.cs

```diff
@@ -383,6 +387,29 @@ public Task TransformAsync(OpenApiOperation operation, OpenApiOperationTransform
 
             return Task.CompletedTask;
         }
+
+        private static OpenApiParameter UnwrapOpenApiParameter(IOpenApiParameter sourceParameter)
```

Comment:
I like this - I might borrow it for Swashbuckle 🙂

Reply:
Hmmm -- I wonder if it might be worth integrating into the Microsoft.OpenAPI APIs? Let me know how it shakes out for you in practice and we can consider moving it to the base library?